insights/parsers/crontab.py

```diff
@@ -147,7 +147,6 @@ def parse_content(self, content):
             cron_match = cron_re.match(line)
             env_match = env_re.match(line)
             if cron_match:
-                print "cron match:", line
                 self.data.append(cron_match.groupdict())
             elif env_match:
                 print "env match:", line
```

Comment:
Here is another `print` you might forget.

Reply:
I think I fixed that in a recent update...

Reply:
Hey @PaulWay the `print` on line 152 is still there.